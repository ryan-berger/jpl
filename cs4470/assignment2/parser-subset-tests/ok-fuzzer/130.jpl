read image ">t" to j
return 16
show h3H.9T.
write image t7e_ to ""
