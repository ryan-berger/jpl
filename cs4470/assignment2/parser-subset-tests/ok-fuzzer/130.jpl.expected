(ReadImageCmd ">t" (VarArgument j))
(StmtCmd (ReturnStmt (IntExpr 16)))
(ShowCmd (VarExpr h3H.9T.))
(WriteImageCmd (VarExpr t7e_) "")
